src/__tests__/use-app/__fixtures__/custom-app-with-gip-custom-props/pages/_app.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import App from 'next/app';
+
+export default function CustomApp({ Component, pageProps, customProps }) {
+  return (
+    <>
+      _app
+      <Component {...pageProps} />
+      {customProps && 'ok'}
+    </>
+  );
+}
+
+CustomApp.getInitialProps = async (appContext) => {
+  const appProps = await App.getInitialProps(appContext);
+  const customProps = true;
```

Comment:
What about `customProps` -> `customAppProps`? It took me a while to understand who was actually providing the custom prop :)

Reply:
changed to `appCustomProps`